lib/identity_config.rb

```diff
@@ -352,6 +352,7 @@ def self.build_store(config_map)
     config.add(:password_pepper, type: :string)
     config.add(:personal_key_retired, type: :boolean)
     config.add(:phone_carrier_registration_blocklist, type: :comma_separated_string_list)
```

Comment:
worth removing the old one while we're here?

Reply:
yep, done